crates/streamer/Cargo.toml

```diff
@@ -21,6 +21,7 @@ webm-iterable = { version = "0.6", features = ["futures"] }
 cadeau = { version = "0.5", features = ["dlopen"] }
 thiserror = "2"
 num_cpus = "1.16.0"
+serde_json = "1.0.133"
```

Comment:
question: Where are you using this new dependency?

Reply:
updated